src/test/java/org/broadinstitute/hellbender/tools/walkers/fasta/ShiftFastaTest.java

```diff
@@ -0,0 +1,50 @@
+package org.broadinstitute.hellbender.tools.walkers.fasta;
+
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.broadinstitute.hellbender.GATKBaseTest;
+import org.broadinstitute.hellbender.testutils.ArgumentsBuilder;
+import org.broadinstitute.hellbender.testutils.BaseTest;
+import org.broadinstitute.hellbender.testutils.FastaTestUtils;
+import org.testng.annotations.Test;
+
+import java.io.File;
+
+public class ShiftFastaTest extends CommandLineProgramTest {
+
+    private static final File MITO_REF = new File(toolsTestDir, "mutect/mito/Homo_sapiens_assembly38.mt_only.fasta");
+    private static final File SHIFTED_MITO_REF = new File(largeFileTestDir + "mitochondria_references/Homo_sapiens_assembly38.chrM.shifted_by_8000_bases.fasta");
```

Comment:
Can you make sure this is properly checked in? It may be an issue with github display but I don't see those files being checked in for this PR so you may have forgotten to add them properly through lfs.